RetailCoder.VBE/UI/Refactorings/AssignedByValParameterQuickFixMockDialogFactory.cs

```diff
@@ -0,0 +1,20 @@
+﻿
+namespace Rubberduck.UI.Refactorings
+{
+    public class AssignedByValParameterQuickFixMockDialogFactory : IAssignedByValParameterQuickFixDialogFactory
```

Comment:
We don't need to make actual mock implementations to inject in the tests - we use a mocking framework for that (Moq).